hw2/03-count.html

```diff
@@ -1,24 +1,27 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
Not a major thing but if possible try giving a bit darker yellow color for highlighting so that it is clearly visible.

Reply:
Thanks, I also found it to be kind of hard to see. I'll look into how to use a span and style it so it can be more visible.